src/bandersnatch/tests/test_verify.py

```diff
@@ -193,3 +194,7 @@ async def test_metadata_verify(monkeypatch):
     monkeypatch.setattr(bandersnatch.verify, "delete_unowned_files", do_nothing)
     monkeypatch.setattr(bandersnatch.verify.os, "listdir", some_dirs)
     await metadata_verify(fc, fa)
+
+
+if __name__ == "__main__":
+    pytest.main(sys.argv)
```

Comment:
I generally just run `pytest src/bandersnatch/tests/test_verify.py` to run specific tests ...